### ISSUE TITLE
Augementation applied on validation data 

### ISSUE THREAD

**Issue by skaliy** [ruescog/semantic_segmentation_augmentations#5]
Great work! However, the augmentations are applied on the validation data, which makes the assessment during the training slightly more difficult. Perhaps, you could implement the augmentations as in fastai.

**Comment by ruescog** [comment]
Hi @skaliy , thank you for your feedback.

In fastai's documentation (https://docs.fast.ai/callback.core.html), it is said that the _before_batch_ part is applied:

> called at the beginning of each batch, just after drawing said batch. It can be used to do any setup necessary for the batch (like hyper-parameter scheduling) or to change the input/target before it goes in the model (change of the input with techniques like mixup for instance).

Maybe I didn't understand you correctly or that's not what you meant, can you explain it a little?

**Comment by skaliy** [comment]
Was thinking about a flag for `run_valid = False`, so the augmentations are not applied to the validation data.

**Comment by ruescog** [comment]
Hi again @skaliy ,

you are precisely right. Augmentations applied on validation set can make the test results no comparable and, as you said, harder to the network.

We will implement it as soon as possible.
